internal/service/certificatesigningrequest.go

```diff
@@ -56,9 +57,16 @@ func signApprovedCertificateSigningRequest(ca *crypto.CA, request api.Certificat
 
 // (DELETE /api/v1/certificatesigningrequests)
 func (h *ServiceHandler) DeleteCertificateSigningRequests(ctx context.Context, request server.DeleteCertificateSigningRequestsRequestObject) (server.DeleteCertificateSigningRequestsResponseObject, error) {
+	allowed, err := auth.GetAuthZ().CheckPermission(ctx, "certificatesigningrequests", "deletecollection")
+	if err != nil {
+		return server.DeleteCertificateSigningRequests503JSONResponse{Message: fmt.Sprintf("auth failed: %v", err)}, nil
```

Comment:
Can you please also update the error message ("Authorisation server unavailable") and not return the `err`?
Instead, we probably want to log the error.